### PR TITLE
[PM-8386] - Add section header to import page

### PR DESCRIPTION
**Pull request by jaasen-livefront**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/PM-8386

## 📔 Objective

This PR updates the import page to group the inputs into sections with a corresponding header.

** **NOTE** ** Adjustments to the background color was required for all clients in order to allow for a contrast between the card and background.

## Desktop

|Light|Dark|
---|---
|desktop-light|desktop-dark|

## Web
|Light|Dark|
---|---
|web-light|web-dark|

## Browser
|Light|Dark|
---|---
|Screenshot 2024-07-17 at 12 49 24 PM|Screenshot 2024-07-17 at 12 49 02 PM|

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: apps/browser/src/_locales/en/messages.json [hunk 1]
```diff
@@ -3087,7 +3087,7 @@
     "message": "Invalid file password, please use the password you entered when you created the export file."
   },
   "importDestination": {
```
Comment (djsmith85) [inline apps/browser/src/_locales/en/messages.json:3089]: Any change of message content without changing the key won't be reflected in existing translations. Please change the key and replace it's usages. This needs to be done for all clients.
```suggestion
  "destination": {
```

**Review by willmartian — CHANGES_REQUESTED** [review]
This PR changes the bg color of `bit-layout` and `bit-dialog` from `tw-bg-background` to `tw-bg-background-alt`. This will affect _all_ pages and dialogs, and is not behind feature flag.

Is that desired, @danielleflinn ?

**Comment by danielleflinn** [comment]
> This PR changes the bg color of` bit-layout` and `bit-dialog` from` tw-bg-background` to `tw-bg-background-alt`. This will affect all pages and dialogs, and is not behind feature flag.

No we should not do this everywhere. I think the background color changes need to be implemented as variants of these components for now since there are many layouts/dialogs in the web app that have not been designed to display on the darker background. 

Perhaps for now, we remove just the background colors from this work and add a follow on task to come back once we have a better plan to support this? @willmartian @djsmith85 does that work for you? Or should we put this task on hold?

**Comment by jaasen-livefront** [comment]
@danielleflinn @willmartian Sounds good. I'll revert the changes to the background.

**Review by djsmith85 — APPROVED** [review]

**Comment by jaasen-livefront** [comment]
@danielleflinn @willmartian Ready for re-review

**Comment by willmartian** [comment]
> @willmartian please advise on what should be changed in the Dialog UI

I have [a PR](https://github.com/bitwarden/clients/pull/10237) open to add a background input to the dialog component. Y'all can either go ahead and add `background="alt"` to the relevant dialogs, or wait for that to be merged and pull it in.

**Review by danielleflinn — APPROVED** [review]

**Pull request merged**